### ISSUE TITLE
RFC: The next meetup 2015/11/03

### ISSUE THREAD

**Issue by Byron** [Byron/rust-hack-and-learn#2]
Hi, 

This ticket should contain a rough sketch on how the next meetup should be looking like.
# tl;dr

I like the format, and want to use our focus time to show how important social aspects are to our group by contrasting a bad pairing experience with a good one. Pairing = social practice.

What do you think about that ?
# The long Story

Personally, I liked the previous format and would like to stick with it, so there are 30m of getting ready, which probably split into 15m of our audience getting ready for input, and another 15m of unshared attention focussed on the guy in front.

It really is these 15m I'd call: Focus Time, as we can use it to emphasize a single aspect which is important to us. For this meetup, I feel this should be what really sets us apart: _embracing the fact the first and foremost, we are social beings_. 

We are social as we treat each other respectfully. This allows us to pair with people we have just met, get to know them, and increase chances of setting up connections which even last past the meetup.  To me it seems like one major component needed to create a community.

Pairing seems to be the way to practice social, as it gently nudges everyone to leave their comfort zone and interact with someone else. This should be a pleasurable experience for everyone, which might require some amount of coaching. 

To achieve that, I'd like to spend the 15m of focus time with showing how a nice pairing experience can look like, and possibly even contrast it with a (dramatized) one that shows how it can go wrong.

What do you think about this proposition ?
